x-pack/plugins/ml/public/data_frame/components/group_by_list/popover_form.test.tsx

```diff
@@ -80,7 +80,7 @@ describe('Data Frame: Group By <PopoverForm />', () => {
       aggName: 'the-agg-name',
       dropDownName: 'the-drop-down-name',
       field: 'the-field',
-      interval: '10m',
+      calendar_interval: '10m',
```

Comment:
As above. seeing as you can only specify `1m`, `1h` etc in the UI, would `1m` rather than `10m` be a better test?

Reply:
Fixed in 71c2346.